docker/Makefile

```diff
@@ -7,7 +7,7 @@ NETCDF-JAR := netcdfAll-5.0.0-SNAPSHOT.jar
 GDAL-BLOB := archives/gdal-and-friends.tar.gz
 PYTHON-BLOB := archives/geopyspark-and-friends.tar.gz
 GEONOTEBOOK := 178bde1679ae3bc084c044ca3eeb915a137f689f
```

Comment:
The `GEONOTEBOOK` SHA should be updated to `eeb1ee9e2dfb9df84eec24306c4b05b7008865ec` which is the merge of renamed changes: https://github.com/geotrellis/geonotebook/commit/eeb1ee9e2dfb9df84eec24306c4b05b7008865ec